specification/electronic-prescription-service-api.yaml

```diff
@@ -348,19 +391,146 @@ paths:
+                    $ref: components/examples/example-a-validation-error-missing-field/Response-FhirError.json
+  /FHIR/R4/$process-message#dispense-claim:
+    post:
+      operationId: send-dispense-claim-message
+      summary: Claim for a dispensed prescription
+      description: |
+        ## Overview
+        Use this endpoint to send a dispense-claim message to EPS.
+
+        The effect of calling this endpoint depends on the type of the message, which is determined by the `MessageHeader` resource.
+        To submit a claim for a dispensed prescription, send a [dispense-claim message](https://simplifier.net/guide/DigitalMedicines/NHSDigital-Claim).
+        
+        Note that the `#dispense-claim` in the URL has been added to distinguish this from the other operations which
+        can be performed using the `$process-message` endpoint. It is not required in requests to the API.
+      tags:
+        - dispensing
+      parameters:
+        - $ref: '#/components/parameters/BearerAuthorization'
+        - $ref: '#/components/parameters/RequestID'
+        - $ref: '#/components/parameters/CorrelationID'
+      requestBody:
+        required: true
+        content:
+          #FIXME - should be set to application/fhir+json but we have had to change it due to an Apigee SmartDocs bug.
+          application/json:
+            schema:
+              type: object
+            examples:
+              dispense-claim-success:
+                description: |
+                  A valid secondary care dispense-claim message.
+                value:
+                  $ref: components/examples/secondary-care/homecare/acute/nominated-pharmacy/clinical-practitioner/4-Process-Request-Claim-200_OK.json
+        description: ''
+      responses:
+        '200':
+          description: Successfully submitted.
+          content:
+            application/fhir+json:
+              schema:
+                type: object
+              examples:
+                operation-outcome-success:
+                  description: |
+                    A successful response to a dispense-claim request.
+                  value:
+                    $ref: components/examples/operation-outcome-success.json
+        '4XX':
+          description: |
+            An error occurred as follows:
+
+            | HTTP status | Error code  | Description      |
+            | ----------- | ----------- | ---------------- |
+            | 400         | BAD_REQUEST | Invalid request. |
+          content:
+            application/fhir+json:
+              schema:
+                type: object
+              examples:
+                OperationOutcome:
+                  description: |
+                    An error response.
+                  value:
+                    $ref: components/examples/example-a-validation-error-missing-field/Response-FhirError.json
+  /FHIR/R4/Task#return:
+    post:
+      operationId: return
+      summary: Return a downloaded prescription
```

Comment:
is this summary useful for someone reading for the first time?

Reply:
maybe something along the lines of: 'return a downloaded prescription to ~spine~ so another dispenser can dispense'

Reply:
rejig the words not the best example

Reply:
I'd prefer to keep the description brief as this appears in the left nav. The first sentence of the description explains the interaction in more detail.